course/styles/dark.json

```diff
@@ -199,11 +207,61 @@
 			},
 			"core/post-content": {
 				"typography": {
-					"lineHeight": "132%"
+					"lineHeight": "145%"
+				}
+			},
+			"core/post-title": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--x-large)",
+					"letterSpacing": "-0.01em"
+				}
+			},
+			"sensei-lms/course-title" : {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--small)",
+					"fontWeight": "500",
+					"letterSpacing": "-0.01em"
+				}
+			},
+			"sensei-lms/course-theme-course-progress-counter" : {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"fontSize": "clamp(1.125rem, 1.125rem, 1.125rem)",
+					"letterSpacing": "0"
 				}
+			},
+			"sensei-lms/course-theme-lesson-module": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"letterSpacing": "0"
+				}
+			},
+			"sensei-lms/exit-course": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"fontSize": "clamp(1.125rem, 1.125rem, 1.125rem)",
+					"letterSpacing": "0"
+				}
+			},
+			"sensei-lms/page-actions": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"fontSize": "clamp(1.125rem, 1.125rem, 1.125rem)",
```

Comment:
Same as above

Reply:
I've tried but it gets converted to `clamp(0.875rem, 0.875rem + ((1vw - 0.48rem) * 0.481), 1.125rem)`  automatically. 😞

Reply:
I see that 1.125rem is used in multiple places (I've found 3), maybe you can consider adding a custom fontsize like this
![image](https://github.com/Automattic/themes/assets/6820724/77723e65-328a-4e13-9eaf-dafa83f9eccc)

And then use that fontsize using `var(--wp--custom--typography--font-sizes--<Your-Font-Size-Name>)`

I tried it locally and it worked for me.

Reply:
Makes sense. I've added a new custom font size in 731aaa13b417c9063b65a9ad3b5d378ceb44416f. 👍